vault/expiration.go

```diff
@@ -841,11 +841,18 @@ type leaseEntry struct {
 }
 
 // encode is used to JSON encode the lease entry
-func (l *leaseEntry) encode() ([]byte, error) {
-	return json.Marshal(l)
+func (le *leaseEntry) encode() ([]byte, error) {
+	return json.Marshal(le)
 }
 
-func (le *leaseEntry) renewable() error {
+func (le *leaseEntry) renewable() bool {
+	if err := le.renewableErr(); err == nil {
+		return true
+	}
+	return false
+}
+
+func (le *leaseEntry) renewableErr() error {
```

Comment:
I thought this was a bit confusing and was just consolidating the logic for dealing with the error, which the expiration manager uses, and returning the value of renewable.